packages/rancher-grafana/package.yaml

```diff
@@ -1,4 +1,4 @@
 url: https://github.com/grafana/helm-charts.git
 subdirectory: charts/grafana
-commit: 837b4ace32278eee14310f1d654d651d0898bb19
+commit: bf7e1110a5ee9258190d0377fea319bb8e764e62
```

Comment:
this commit is used in Grafana helm chart starting at 6.12.0 https://github.com/grafana/helm-charts/commit/bf7e1110a5ee9258190d0377fea319bb8e764e62

and Grafana chart 6.12.0 uses Grafana image tag 8.0.0

But the original issue says we want to use chart 6.11.0 https://github.com/rancher/rancher/issues/33210

@aiyengar2 Can you double-check the versions?

Reply:
@jiaqiluo Grafana is 6.11.0 in this commit:
https://github.com/grafana/helm-charts/blob/bf7e1110a5ee9258190d0377fea319bb8e764e62/charts/grafana/Chart.yaml#L3-L4.

We don't want to upgrade to 8.x.x since >7.x.x is APGLv3 instead of Apache licensed.